hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneClientAdapterImpl.java

```diff
@@ -707,17 +707,27 @@ public SnapshotDiffReport getSnapshotDiffReport(Path snapshotDir,
       if (takeTemporaryToSnapshot || takeTemporaryFromSnapshot) {
         OFSPath snapPath = new OFSPath(snapshotDir.toString(), config);
         if (takeTemporaryToSnapshot) {
-          objectStore.deleteSnapshot(snapPath.getVolumeName(),
-              snapPath.getBucketName(), toSnapshot);
+          deleteSnapshot(toSnapshot, snapPath);
         }
         if (takeTemporaryFromSnapshot) {
-          objectStore.deleteSnapshot(snapPath.getVolumeName(),
-              snapPath.getBucketName(), fromSnapshot);
+          deleteSnapshot(fromSnapshot, snapPath);
         }
       }
     }
   }
 
+  private void deleteSnapshot(String snapshot, OFSPath snapPath) {
```

Comment:
Should this be added to [OzoneClientUtils](https://github.com/apache/ozone/blob/master/hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/OzoneClientUtils.java#L55) and reuse in `BasicOzoneClientAdapterImpl` and `BasicRootedOzoneClientAdapterImpl`?

Reply:
Done.